### ISSUE TITLE
[Bug]  Error in repeatable Field "Uncaught TypeError: Cannot read properties of undefined (reading 'subfieldsCallbacks')"

### ISSUE THREAD

**Issue by informatico2** [Laravel-Backpack/CRUD#4501]
# Bug report

### What I did

I made multiple repeatable fields in my controller like this:

`$this->crud->addField([   // repeatable
            'name'  => 'schedule_monday',
            'label' => 'Horario Lunes',
            'type'  => 'repeatable',
            'fields' => [
                
                [
                    'name' => 'type',
                    'label' => 'Opción',
                    'type' => 'select_from_array',
                    'options'     => ['0' => 'Abierto', '1' => 'Cerrado', '2' => "24h"],
                    'wrapper' => ['class' => 'form-group col-md-4'],
                    'attributes' => ['id' => 'monday'],
                    'default'     => '0',

                ],
                [
                    'name'     => 'start', // JSON variable name
                    'label'    => "Entrada", // human-readable label for the input
                    'type'     => "time",
                    'wrapper' => ['class' => 'form-group col-md-4', 'id' => 'monday'],// show the field, but don't store it in the database column above
                ],

                [
                    'name'     => 'end', // JSON variable name
                    'label'    => "Salida", // human-readable label for the input
                    'type'     => "time",
                    'wrapper' => ['class' => 'form-group col-md-4', 'id' => 'monday'], // show the field, but don't store it in the database column above
                ],
            ],
        
            // optional
            
            'new_item_label'  => 'Añadir Horario', // customize the text of the button
            'init_rows' => 1, // number of empty rows to be initialized, by default 1 // minimum rows allowed, when reached the "delete" buttons will be hidden
            'max_rows' => 4, // maximum rows allowed, when reached the "new item" button will be hidden
            'reorder' => true, // show up&down arrows next to each row
            'reorder' => 'order',
            'min_rows' => 1, // maximum rows allowed, when reached the "new item" button will be hidden
            'wrapper' => ['class' => 'form-group col-md-12', 'id' => 'parent_monday'],

        
        ]);

        

        $this->crud->addField([   // repeatable
            'name'  => 'schedule_tuesday',
            'label' => 'Horario Martes',
            'type'  => 'repeatable',
            'fields' => [
                
                [
                    'name' => 'type',
                    'label' => 'Opción',
                    'type' => 'select_from_array',
                    'options'     => ['0' => 'Abierto', '1' => 'Cerrado', '2' => "24h"],
                    'wrapper' => ['class' => 'form-group col-md-4'],
                    'attributes' => ['id' => 'tuesday'],
                    'default'     => '0',

                ],
                [
                    'name'     => 'start', // JSON variable name
                    'label'    => "Entrada", // human-readable label for the input
                    'type'     => "time",
                    'wrapper' => ['class' => 'form-group col-md-4', 'id' => 'tuesday'], // show the field, but don't store it in the database column above
                ],

                [
                    'name'     => 'end', // JSON variable name
                    'label'    => "Salida", // human-readable label for the input
                    'type'     => "time",
                    'wrapper' => ['class' => 'form-group col-md-4', 'id' => 'tuesday'], // show the field, but don't store it in the database column above
                ],
            ],
        
            //optional
            
            // 'new_item_label'  => 'Añadir Horario', // customize the text of the button
            // 'init_rows' => 1, // number of empty rows to be initialized, by default 1 // minimum rows allowed, when reached the "delete" buttons will be hidden
            // 'max_rows' => 4, // maximum rows allowed, when reached the "new item" button will be hidden
            // 'reorder' => true, // show up&down arrows next to each row
            // 'reorder' => 'order',
            // 'min_rows' => 1, // maximum rows allowed, when reached the "new item" button will be hidden
            // 'wrapper' => ['class' => 'form-group col-md-12', 'id' => 'parent_tuesday'],

        
        ]);`

### What happened

Uncaught TypeError: Cannot read properties of undefined (reading 'subfieldsCallbacks')
    at setupFieldCallbacks (create:1725:53)
    at newRepeatableElement (create:1551:13)
    at HTMLButtonElement.<anonymous> (create:1445:17)
    at HTMLButtonElement.dispatch (bundle.js?v=5.1.4@4539c787a2e052d2e2e2c00188c38432dc2a7ea9:2:138508)
    at m.handle (bundle.js?v=5.1.4@4539c787a2e052d2e2e2c00188c38432dc2a7ea9:2:136503)
    at Object.trigger (bundle.js?v=5.1.4@4539c787a2e052d2e2e2c00188c38432dc2a7ea9:2:167056)
    at HTMLButtonElement.<anonymous> (bundle.js?v=5.1.4@4539c787a2e052d2e2e2c00188c38432dc2a7ea9:2:167652)
    at Function.each (bundle.js?v=5.1.4@4539c787a2e052d2e2e2c00188c38432dc2a7ea9:2:98111)
    at T.fn.init.each (bundle.js?v=5.1.4@4539c787a2e052d2e2e2c00188c38432dc2a7ea9:2:96590)
    at T.fn.init.trigger (bundle.js?v=5.1.4@4539c787a2e052d2e2e2c00188c38432dc2a7ea9:2:167627)

### Is it a bug in the latest version of Backpack?

After I run ```composer update backpack/crud``` the bug... is it still there?
Yes

### Backpack, Laravel, PHP, DB version

When I run ```php artisan backpack:version``` the output is:
php 7.4, backpack 5.1

**Comment by bartjuh4** [comment]
I'm experiencing this same bug, with the repeatable fields:

### PHP VERSION:
PHP 8.1.2

### LARAVEL VERSION:
v9.19.0@bbce25bd823133f6a5a724f2d62680b711f1d0df

### BACKPACK VERSION:
5.1.4@4539c787a2e052d2e2e2c00188c38432dc2a7ea9

**Comment by pxpm** [comment]
@informatico2 and @bartjuh4 thanks for the report. 

From the error you report you guys don't have the `window.crud` object somehow. 
For the callbacks we define it  here:
https://github.com/Laravel-Backpack/CRUD/blob/a06d04e7ba62a56540f3dd72199f36e2c62ca8a7/src/resources/views/crud/inc/form_fields_script.blade.php#L170

and if I `console.log(window.crud)` in the `setupFieldCallbacks` in `repeatable.blade.php` I have the object that we defined:

![image](https://user-images.githubusercontent.com/7188159/178246964-3ed13bd9-7c5a-4b7b-95d2-3ca5fc3e79d3.png)

So I will take the guess that **both of you are overriding the `form_content.blade.php` file**, if that's the case please update the file to have the scripts `@include('crud::inc.form_fields_script')` have a look here: https://github.com/Laravel-Backpack/CRUD/blob/a06d04e7ba62a56540f3dd72199f36e2c62ca8a7/src/resources/views/crud/form_content.blade.php#L189

Let me know how it goes.

Cheers

**Comment by tabacitu** [comment]
Good catch @pxpm ! We should add this to the v5 upgrade docs, so people who are upgrading know they should add this to their `form_content.blade.php` file, if they've modified it.

**Comment by promatik** [comment]
@bartjuh4 are you using the latest version of `backpack-pro`?
I'm asking this because we now have;

```js
if(!subfield.input?.getAttribute('change-event-applied')) {
```

Note the `?.getAttribute`, it may avoid the error you have.
If it doesn't solve your issue, please let us know, probably by opening a new issue 🙌
